src/lightning/pytorch/_graveyard/__init__.py

```diff
@@ -11,4 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import lightning.pytorch._graveyard.hpu.accelerator
+import lightning.pytorch._graveyard.hpu.io_plugin
+import lightning.pytorch._graveyard.hpu.precision_plugin
+import lightning.pytorch._graveyard.hpu.strategy_parallel
+import lightning.pytorch._graveyard.hpu.strategy_single
```

Comment:
I suggest putting all these classes together in one file for simplicity

Reply:
how then assign `self` would work?

Reply:
Here's one example: https://github.com/Lightning-AI/lightning/blob/1.9.5/src/pytorch_lightning/_graveyard/training_type.py#L24-L40

Reply:
And its tests: https://github.com/Lightning-AI/lightning/blob/1.9.5/tests/tests_pytorch/graveyard/test_training_type.py